daprdocs/content/en/developing-applications/develop-components/pluggable-components/pluggable-components-sdks/_index.md

```diff
@@ -0,0 +1,19 @@
+---
+type: docs
+title: "Pluggable components SDKs"
+linkTitle: "SDKs"
+weight: 2000
+description: "Develop pluggable components in your favorite language"
+no_list: true
+is_preview: true
+---
+
+The Dapr SDKs are the easiest way for you to create Dapr Pluggable Components. Choose your favorite language and get up and running with Dapr in minutes.
```

Comment:
```suggestion
The Dapr SDKs are the easiest way for you to create pluggable components. Choose your favorite language and get up and running creating components in minutes.
```

Reply:
Generally agree, but I find "get up and running creating components" a bit awkward still.  How about:

*The Dapr SDKs are the easiest way for you to create pluggable components. Choose your favorite language and start creating components in minutes.*